tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -540,3 +547,46 @@ export function getSingleLineExcerptText(excerpt: Excerpt): string {
 	// Regex replaces line breaks with spaces to ensure everything ends up on a single line.
 	return excerpt.text.trim().replace(/\s+/g, " ");
 }
+
+/**
+ * Resolves a symbolic link and creates a URL to the target.
+ *
+ * @param contextApiItem - See {@link TsdocNodeTransformOptions.contextApiItem}.
+ * @param codeDestination - The link reference target.
+ * @param apiModel - The API model to which the API item and destination belong.
+ *
+ * @throws If the reference cannot be resolved.
+ */
+export function resolveSymbolicReference(
```

Comment:
New function, derived from existing `resolveSymbolicLink` function under `api-item-transforms/Utilities.ts`. That function now leverages this more general one.